src/dialog/index.ts

```diff
@@ -1,6 +1,20 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+
+import {
+  Widget
+} from 'phosphor/lib/ui/widget';
+
+import {
+  Panel
+} from 'phosphor/lib/ui/panel';
+
+import {
+  Message
+} from 'phosphor/lib/core/messaging';
```

Comment:
In general, please `import` in alphabetical order based on path. So these three should be: 

* `phosphor/lib/core/messaging`
* `phosphor/lib/ui/panel`
* `phosphor/lib/ui/widget`